x-pack/metricbeat/module/mssql/_meta/docs.asciidoc

```diff
@@ -0,0 +1,19 @@
+This is the https://www.microsoft.com/en-us/sql-server/sql-server-2017[Microsoft SQL 2017] Metricbeat module. It is still in beta and under active development to add new Metricsets and introduce enhancements.
+
+The idea is to improve the current set of metrics by fetching data from https://docs.microsoft.com/en-us/sql/relational-databases/system-dynamic-management-views/system-dynamic-management-views?view=sql-server-2017[System Dynamic Management Views]. A curated dashboard will also be included soon.
```

Comment:
I think this info belongs into the Meta Github issue and not the docs. The docs should only contain what we have not what we plan.